src/core/raster/qgsrasterfilewriter.cpp

```diff
@@ -1072,6 +1072,15 @@ QString QgsRasterFileWriter::driverForExtension( const QString &extension )
   if ( ext.startsWith( '.' ) )
     ext.remove( 0, 1 );
 
+  if ( ext == QLatin1String( "tif" ) || ext == QLatin1String( "tiff" ) )
```

Comment:
Shouldn't the comparison be case insensitive?

Reply:
indeed. Fixes. Thanks for the review